src/ws/remotectl-certificate.c

```diff
@@ -106,16 +107,23 @@ set_cert_attributes (const gchar *path,
         }
     }
 
+  if (g_stat (path, &statbuf) < 0)
+    {
+      g_message ("could not stat certificate: %s: %s", path, g_strerror (errno));
+      goto out;
+    }
+
```

Comment:
this is getting out of hand....

Could we just set the correct permissions when creating the certificate and remove all of this extra logic?

If the admin went out of their way to change the permissions, who are we to say otherwise?

Reply:
Good point -- I'll look into that. Thanks!